app/src/main/java/com/sunlightlabs/congress/services/RollService.java

```diff
@@ -43,18 +45,25 @@
     public static final String VOTE_TWO_THIRDS  = "2/3 YEA-AND-NAY";
 
 
+    // /{chamber}/votes/recent.json
+    public static List<Roll> latestVotes(int page) throws CongressException {
+        String[] house = { "house", "votes", "recent" };
+        String[] senate = { "senate", "votes", "recent" };
+        List<Roll> houseVotes = rollsFor(ProPublica.url(house, page));
+        List<Roll> senateVotes = rollsFor(ProPublica.url(senate, page));
 
-    public static String[] basicFields = {
-		"roll_id", "chamber", "number", "year", "congress", "bill_id",
-		"bill.official_title", "bill.short_title",
-		"voted_at", "vote_type", "roll_type", "question", "required", "result",
-		"breakdown"
-	};
+        List<Roll> votes = new ArrayList<Roll>();
+        votes.addAll(houseVotes);
+        votes.addAll(senateVotes);
 
-    public static List<Roll> latestVotes(int page, int per_page) throws CongressException {
-        Map<String,String> params = new HashMap<String,String>();
-        params.put("order", "voted_at");
-        return sunlightRollsFor(Congress.url("votes", basicFields, params, page, per_page));
+        Collections.sort(votes, new Comparator<Roll>() {
+            @Override
+            public int compare(Roll lhs, Roll rhs) {
+                return rhs.voted_at.compareTo(lhs.voted_at);
+            }
+        });
+
+        return votes;
```

Comment:
For now, votes from the House and Senate are fetched separately and then merged/sorted client-side. I might split this into two tabs later, see #724.